src/checker.py

```diff
@@ -68,6 +69,10 @@ def _get_finish_args_extra_data_from_json(self, json_data):
     def _get_module_data_from_json(self, json_data):
         external_data = []
         for module in json_data.get('modules', []):
+            # This is a guess at the package name from the name the author
+            # has given to the module block
+            pkg_name = module.get('name', None)
```

Comment:
I think it makes more sense to have the package name as part of the checker data, this way we can enforce it for the CVE Checker.

Reply:
Done